### DISCUSSION TITLE
Why use Landscapist over Coil?

### DISCUSSION THREAD

**Discussion by erawhctim** [skydoves/landscapist#174] [General]
I'm new to your library, and it's not immediately clear why Coil support in landscapist exists while Coil also has support for Jetpack Compose on its own.

**Comment by skydoves** [comment]
Hey @erawhctim, that's a good question!
Check out this thread: https://twitter.com/Louis_CAD/status/1552950792772100096

**Comment by erawhctim** [comment]
Thanks for the link. So Landscapist is really more of a companion library _on top_ of Coil with extra features? Am I thinking about that in the right way?

**Reply by skydoves** [reply]
Essentially the `Landscapist-Coil` is built with top of the core system of Coil, such as network image requests, and gif supports. But the Jetpack Compose implementation is quite different. 

Landscapist is focusing on supporting more pluggable features, such as placeholder, animation, preview, and palette supports by providing plugins. Also, recently I'm focusing on improving the recomposition performance in Jetpack Compose and more general/comprehensive features in image loading that could be used with other image libraries (Glide and Fresco.)

So you can say Jetpack Compose implementation is quite different in many manners.
